runtime/directives/sky-app-link.directive.spec.ts

```diff
@@ -8,24 +8,34 @@ import { SkyAppConfig } from '../config';
 import { SkyAppLinkDirective } from './sky-app-link.directive';
 
 @Component({
-  template: `<a skyAppLink="test">Test</a>`
+  template: `<a skyAppLink='test'>Test</a>`
```

Comment:
No functional difference, but any reason these are single quotes?  Same question for line 16.

Reply:
There's no particular reason. It's probably just muscle memory typing things in a TS file vs. html. That particular line got changed because I copied a bit of code from the external link directive tests and when I massaged it to fit this context, that single quote change was all that was left.

I prefer the double quotes, so I've updated the PR to clean that up plus some other development remnants in the spec file for both tests.